com.prateekmedia.appimagepool.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: com.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=home
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=com.prateekmedia.appimagepool
+modules:
+  - name: appimagepool
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - ls appimagepool
+      - install -Dm644 appimagepool/appimagepool /app/appimagepool/appimagepool
+      - cp -R appimagepool/data /app/appimagepool/data
+      - cp -R appimagepool/lib /app/appimagepool/lib
+      - chmod +x /app/appimagepool/appimagepool
+      - ls /app
+      - chmod +x /app/appimagepool
+      - install -Dm644 appimagepool/appimagepool.svg /app/share/icons/hicolor/scalable/apps/appimagepool.svg
+      - mkdir /app/bin
+      - ln -s /app/appimagepool/appimagepool /app/bin/appimagepool
+      - install -Dm644 appimagepool/appimagepool.desktop /app/share/applications/appimagepool.desktop
+      - install -Dm644 com.prateekmedia.appimagepool.appdata.xml /app/share/metainfo/com.prateekmedia.appimagepool.appdata.xml
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
```

Comment:
while it would be better to also have a `aarch64`, if you only have `x86_64` you must disable `aarch64`. See https://github.com/flathub/flathub/wiki/App-Maintenance#dont-build-on-arm for an example (except it's `aarch64`)

Reply:
I think it's only possible with docker, but I don't have it setup on my github workflow, I will add it in future.

Reply:
flathub is designed to build from source, so setting things up to build from source in the manifest would do the trick. That's how most app get built.